Library/ViewModels/ProjectPamphletMainCellViewModel.swift

```diff
@@ -417,3 +436,30 @@ private func progressColor(forProject project: Project) -> UIColor {
     return .ksr_green_700
   }
 }
+
+private func projectCampaignExperiment() -> ProjectCampaignExperimentType {
+  let userAttributes = optimizelyUserAttributes(
+    with: AppEnvironment.current.currentUser,
+    project: nil,
+    refTag: nil
+  )
+
+  let optimizelyVariant = AppEnvironment.current.optimizelyClient?
```

Comment:
If we remove `ProjectCampaignExperimentType` then we can just return this directly as `OptimizelyExperiment.Variant?`.